cluster/saltbase/salt/kube-apiserver/initd

```diff
@@ -38,6 +38,9 @@ DAEMON_USER=kube-apiserver
 #
 do_start()
 {
+        # Raise the file descriptor limit - we expect to open a lot of sockets!
+        ulimit -n 65536
```

Comment:
I mean yeah 2^16 :)
Was hoping for some knowledge about system limits, or what machine size you're targeting with that number.